molecule/default/tests/test_cinder_service.py

```diff
@@ -1,23 +1,30 @@
-import os
+# -*- coding: utf-8 -*-
+# ==============================================================================
+# Imports
+# ==============================================================================
 import pytest
-import testinfra.utils.ansible_runner
-
-testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
-    os.environ['MOLECULE_INVENTORY_FILE']).get_hosts('shared-infra_hosts')[:1]
-
-utility_container = ("lxc-attach -n $(lxc-ls -1 | grep utility | head -n 1) "
-                     "-- bash -c '. /root/openrc ; ")
+import pprint
 
 
+# ==============================================================================
+# Test Cases
+# ==============================================================================
 @pytest.mark.test_id('d7fc5630-432a-11e8-b9da-6a00035510c0')
 @pytest.mark.jira('asc-222')
-def test_cinder_service(host):
-    """Test to verify that cinder service is running
+def test_cinder_service(os_api_conn):
+    """Test to verify that cinder service is enabled
 
     Args:
-        host(testinfra.host.Host): A hostname in
-            dynamic_inventory.json/molecule.yml
+        os_api_conn (openstack.connection.Connection): An authorized API
+            connection to the 'default' cloud on the OpenStack infrastructure.
     """
-    cmd = "{} cinder service-list'".format(utility_container)
-    output = host.run(cmd)
-    assert ("cinder-volume" in output.stdout)
+
+    # Getting the list of services (return list of Munch Python dictionaries)
+    service_list = os_api_conn.list_services()
+    pprint.pprint(service_list)
+
+    filtered_services = list(filter(lambda d: d['name'].lower() == 'cinder',
+                                    service_list))
+    assert len(filtered_services) > 0, 'cinder service is not found'
```

Comment:
Simplified as a Python list of zero length will assert to `False`. Also, enhance the assertion print to give all relevant information.

```suggestion
    assert filtered_services, "cinder service is not found!\n{}".format(pformat(unmunchify(service_list)))
```